web-animations-2/Overview.bs

```diff
@@ -2006,6 +2045,34 @@ partial interface AnimationTimeline {
 
 </div>
 
+<h3 id="the-animation-interface">The <code>Animation</code> interface</h3>
+<pre class="idl">
+[Exposed=Window]
+partial interface Animation : EventTarget {
+    attribute CSSNumberish?       startTime;
+    attribute CSSNumberish?       currentTime;
+};
+</pre>
+
+<div class='attributes'>
```

Comment:
As with the interface, please clarify that these are updates to the two members of the Animation interface.

Reply:
done.